addons/web_unsplash/static/src/xml/unsplash_image_widget.xml

```diff
@@ -27,58 +27,63 @@
 </t>
 
 <t t-name="web_unsplash.dialog.error.credentials">
-    <h4><t t-esc="title"/></h4>
-    <div class="details">
-        <t t-esc="subtitle"/>
-        <div class="form-group mt-4 access_key_box">
-            <input type="text" class="form-control w-100" id="accessKeyInput" placeholder="Paste your access key here"/>
-        </div>
-        <a href="https://www.odoo.com/documentation/user/general/unsplash/unsplash_access_key.html" target="_blank"><i class="fa fa-arrow-right"/> Generate an access key</a>
-        <div class="form-group mt-4 access_key_box">
-            <input type="text" class="form-control w-100" id="appIdInput" placeholder="Paste your application ID here"/>
-        </div>
-        <a href="https://www.odoo.com/documentation/user/general/unsplash/unsplash_application_id.html" target="_blank"><i class="fa fa-arrow-right"/> How to find my Unsplash Application ID?</a>
-        <button type="button" class="btn btn-primary btn-block mt-4 save_unsplash">Apply</button>
+    <div class="alert alert-info w-100">
+        <h4><t t-esc="title"/></h4>
+        <p><t t-esc="subtitle"/></p>
+        <t t-if="showUnsplashSetup">
+            <a href="https://www.odoo.com/documentation/user/general/unsplash/unsplash_access_key.html" target="_blank">Get an Access key</a>
+            and paste it here:
+            <div class="form-group d-inline">
```

Comment:
What's all those uses of d-inline classes? Bootstrap allows to create horizontal forms without using those: https://getbootstrap.com/docs/4.5/components/forms/#inline-forms

Reply:
Yes It sounds a little bit hackish, but it was used to get a clean layout on smaller viewports :

TEXT -> INPUT 
TEXT -> INPUT
BUTTON

instead of :

TEXT -> INPUT TEXT -> INPUT
BUTTON

or :

TEXT -> INPUT TEXT
INPUT -> BUTTON

I'll check if I can get the same thing with form-inline.

Reply:
Website edition from a mobile is not supported anyway ;) but bootstrap is mobile-first so form-inline will work as well